docs/pages/guides/customizing-metro.md

```diff
@@ -26,6 +28,104 @@ You can find a complete list of supported options in the Metro docs: [Configurin
 
 To add to a value, such as an array of file extensions, defined in the default configuration, you can access the defaults using the `getDefaultConfig(projectRoot)` function defined in `@expo/metro-config`. Add `@expo/metro-config` to the dependencies of your project to do this. In SDK +41 you can import from `expo/metro-config`.
 
+## Web Support
+
+> Metro web support is an experimental feature in Expo SDK 46 and only works with the [local Expo CLI](https://blog.expo.dev/new-versioned-expo-cli-cf6e10632656).
+
+By default, Expo CLI uses Webpack as the bundler on web platforms, this is because Metro historically did not support web.
+Using different bundlers across platforms leads to some critical divergence in how your app works across platforms. Features like Fast Refresh which work on native don't work on web, and important production functionality like assets are treated differently across bundlers.
+
+By utilizing Metro across all platforms you can have a more universal development experience. You also get to utilize shared cached chunks across platforms meaning faster iteration speed when working across platforms. Project upgrades can also be easier since there are less dependencies (`webpack`, `webpack-dev-server`) you need to update between versions.
+
+All Metro web features should be universal (bundling, static files, assets) making the DX easier to understand and faster across the app development process.
+
+Learn once, bundle everywhere!
+
+### Expo Webpack vs. Expo Metro
+
+Universal Expo Metro is designed to be fully universal, meaning any web bundling features should also work on native too.
+Because of this we make some breaking changes between the two bundler implementations, carefully check the difference if you're moving from Webpack to Metro.
+
```

Comment:
```suggestion
Universal Expo Metro is designed to be fully universal, meaning any web bundling features should also work on native too. Because of this we make some breaking changes between the two bundler implementations, carefully check the difference if you're moving from Webpack to Metro.

```